tests/Makefile.am

```diff
@@ -59,7 +59,6 @@ dist_check_SCRIPTS = $(srcdir)/glade_tests/*.py \
 		     $(srcdir)/*_tests/*/*/*/*.py \
 		     $(srcdir)/unit_tests/*_tests/*.py \
 		     shellcheck/*.sh \
-		     webui_eslint/run_webui_eslint.sh \
```

Comment:
there is another occurrence of webui_eslint.sh which you forgot to remove.

```
[jelle@t14s][~/projects/anaconda]%git grep run_webui_eslint.sh
tests/Makefile.am:                   webui_eslint/run_webui_eslint.sh \
tests/Makefile.am:      webui_eslint/run_webui_eslint.sh
```

Reply:
Thanks!